MeiliSearchBox.vue

```diff
@@ -207,8 +207,9 @@ dark-input()
     // Footer
     .docs-searchbar-footer
       display flex 
-      justify-content space-between 
-      align-items center 
+      justify-content flex-end 
+      align-items center
+      column-gap 10px
```

Comment:
There is still a gap between the text and the logo. I think if we just remove all these lines, it works

Reply:
It won't work if we remove all these lines. The issue was because of **justify-content space-between**.

The little space is because of **column-gap 10px**. I don't know if it would look good if all the space is removed but if you want 0 space let me know. I'll remove the 10px gap b/w the text and logo.

Reply:
Did you remove `display flex` as well? I tried it locally and it seems to work

Reply:
<img width="626" alt="Screenshot 2022-10-27 at 11 16 21" src="https://user-images.githubusercontent.com/33010418/198244878-ed154485-f9c4-4868-a673-d5f24764dd23.png">
<img width="504" alt="Screenshot 2022-10-27 at 11 16 42" src="https://user-images.githubusercontent.com/33010418/198244880-6534ea1d-0cfa-4c47-af85-e38663c95447.png">

Reply:
Oh, flex was there. I have updated the PR. Thanks.